cpu/esp32/esp-now/esp_now_netdev.c

```diff
@@ -162,12 +162,12 @@ static void IRAM_ATTR esp_now_scan_peers_done (void)
         critical_exit_var(state);
     }
 
-    #if ENABLE_DEBUG
+#if ENABLE_DEBUG
```

Comment:
It is not forbidden to indent a preprocessor directive. The Linux style guide recommends not to use preprocessor directives whenever possible because preprocessor directives make the code harder to read. This is even more true when they are not indented.

Reply:
My IDE keeps doing them somewhat automatically, so factoring them out into their own commit makes them easier to get rid of when they are ultimately unwanted.